### PR TITLE
[REVIEW] CI style check script

### PR DESCRIPTION
**Pull request by mike-wendt**
Script for flake8 testing, adding to repo to allow for additional style checks in the future. A part of the larger effort to move scripts out of CI and into the repo.

### CODE REVIEW

File: ci/checks/style.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```
Comment (kkraus14) [inline ci/checks/style.sh]: I think we should add the NVIDIA copyright below the shebang here.

**Comment by mike-wendt** [comment]
@kkraus14 changes made

**Comment by mike-wendt** [comment]
rerun tests

**Comment by kkraus14** [comment]
cc @raydouglass to review before merging.

**Pull request merged**